charts/argo-rollouts/templates/argo-rollouts-aggregate-roles.yaml

```diff
@@ -2,12 +2,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ .Release.Name }}-aggregate-to-view
+  name: {{ include "argo-rollouts.fullname" . }}-view
```

Comment:
Is there any read we rename the ClusterRole's name? It would be better to make them consistent as the names in argo-rollouts's manifest

https://github.com/argoproj/argo-rollouts/blob/ff65b39f21a0be79744d45d7169de6ee84e991a6/manifests/base/argo-rollouts-aggregate-roles.yaml#L4

Reply:
But are you okay with replacing `.Release.Name` by the template function? I deleted the `aggregate-to-` to reduce the resource name lenght as it is limited. And also because other projects (eg. cert-manager) also use `<template>-view`, `<template>-edit`.

Reply:
Thanks for the explanation. I think it's good to use a template function to format the name automatically.
One suggestion: due to the name change, could you please put some notes in the `## Upgrading` section of *README.md* for the sake of users who have been used to the old names? Thanks.

Reply:
I updated to `{{ include "argo-rollouts.fullname" . }}-aggregate-to-view` etc.
And I also updated the Upgrading section.